src/lib/remove-index-path.ts

```diff
@@ -8,10 +8,15 @@ type Paths = GetStaticPathsResult['paths']
  */
 export function removeIndexPath(paths: Paths): Paths {
 	return paths.filter((pathEntry) => {
-		const isIndexPath =
-			typeof pathEntry == 'string'
-				? pathEntry == ''
-				: pathEntry.params.page.length == 0
+		let isIndexPath = false
+		if (typeof pathEntry == 'string') {
+			isIndexPath = pathEntry == ''
+		} else if (typeof pathEntry == 'object') {
+			// the page array can be empty or contain a single empty string
+			isIndexPath =
+				pathEntry.params.page.length == 0 || pathEntry.params.page[0] == ''
```

Comment:
This is a fix, the vagrant builds were failing because we introduced empty index paths as empty string into the content. We should support this case anyway.